packages/core-v2/src/visual-model/visual-model.ts

```diff
@@ -52,7 +52,7 @@ export interface VisualModel extends LegacyModel {
     /**
      * @returns All stored model color pairs.
      */
-    getModelColors(): Map<string, VisualModelData>;
+    getModelsData(): Map<string, VisualModelData>;
 
```

Comment:
Comment is no longer true, so maybe something like:

`@returns All stored model data pairs`?